lib/zendesk_apps_tools/theme.rb

```diff
@@ -40,8 +40,8 @@ def generate_payload
           asset_payload[File.basename(asset)] = url_for(asset)
         end
         payload['assets'] = asset_payload unless asset_payload.empty?
-        payload['js'] = javascript if javascript
-        payload['css'] = stylesheet_content if stylesheet_content
+        payload['js'] = url_for(theme_package_path('script.js')) if File.file?(theme_package_path('script.js'))
```

Comment:
Good for completeness, but [doesn't look like this is being used](https://github.com/zendesk/help_center/pull/13668/files#diff-127a850b7d7b20173ad3fba88f8a25e3R134) on the HC side.  Same for CSS